bindings/rust/generate/generate-fips.sh

```diff
@@ -29,9 +30,7 @@ source "${SCRIPT_DIR}"/_generation_tools.sh
 
 # Clone the FIPS branch in local.
 function clone_fips_branch {
-  pushd "${TMP_DIR}"
-  git clone -b ${AWS_LC_FIPS_BRANCH} --depth 1 --single-branch https://github.com/awslabs/aws-lc.git
-  popd
+  git clone -b "${AWS_LC_FIPS_GIT_BRANCH}" --depth 1 --single-branch "${AWS_LC_FIPS_GIT_CLONE_URL}" "${AWS_LC_FIPS_DIR}"
 }
```

Comment:
I think it would be safer to give the user an opportunity to sanity check the repo/branch we're cloning:
```
function clone_fips_branch {
  while true; do
    echo Cloning from: "${AWS_LC_FIPS_GIT_CLONE_URL}":"${AWS_LC_FIPS_GIT_BRANCH}"
    read -p "Continue? " yn
    case $yn in
      [Yy]* ) break;;
      [Nn]* ) exit;;
      * ) echo "Please answer yes or no.";;
     esac
  done

  git clone -b "${AWS_LC_FIPS_GIT_BRANCH}" --depth 1 --single-branch "${AWS_LC_FIPS_GIT_CLONE_URL}" "${AWS_LC_FIPS_DIR}"
}
```